extensions/ql-vscode/test/vscode-tests/no-workspace/data-extensions-editor/model-details/model-details-panel.test.ts

```diff
@@ -0,0 +1,98 @@
+import { window, TreeView } from "vscode";
+import { CodeQLCliServer } from "../../../../../src/codeql-cli/cli";
+import {
+  CallClassification,
+  ExternalApiUsage,
+} from "../../../../../src/data-extensions-editor/external-api-usage";
+import { ModelDetailsPanel } from "../../../../../src/data-extensions-editor/model-details/model-details-panel";
+import { DatabaseItem } from "../../../../../src/databases/local-databases";
+import { mockedObject } from "../../../utils/mocking.helpers";
+
+describe("ModelDetailsPanel", () => {
+  const mockCliServer = mockedObject<CodeQLCliServer>({});
+
+  describe("setState", () => {
+    const hideModeledApis: boolean = false;
+    const externalApiUsages: ExternalApiUsage[] = [
+      {
+        library: "test",
+        supported: false,
+        supportedType: "none",
+        usages: [],
+        signature: "test",
+        packageName: "test",
+        typeName: "test",
+        methodName: "test",
+        methodParameters: "test",
+      },
+    ];
+    const dbItem = mockedObject<DatabaseItem>({
```

Comment:
We define the exact same `dbItem` in each `describe` block. Could we move the definition one level higher, like where we make `mockCliServer`?

Reply:
👍